cmd/init.go

```diff
@@ -5,14 +5,88 @@ import (
 
 	"github.com/chanzuckerberg/fogg/errs"
 	fogg_init "github.com/chanzuckerberg/fogg/init"
+	prompt "github.com/segmentio/go-prompt"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
 func init() {
+	initCmd.Flags().String("project", "", "Use this to pass the project name via CLI.")
+	initCmd.Flags().String("region", "", "Use this to pass the aws region via CLI.")
+	initCmd.Flags().String("bucket", "", "Use this to pass the infra bucket name via CLI.")
+	initCmd.Flags().String("table", "", "Use this to pass the infra dynamo table name via CLI.")
+	initCmd.Flags().String("profile", "", "Use this to pass the aws auth profile via CLI.")
+	initCmd.Flags().String("owner", "", "Use this to pass the owner name via CLI.")
 	rootCmd.AddCommand(initCmd)
 }
 
+func isFlagPassed(cmd *cobra.Command, name string) bool {
+	found := false
+	cmd.Flags().Visit(func(f *pflag.Flag) {
+		if f.Name == name {
+			found = true
+		}
+	})
+	return found
+}
+
+func userPrompt(cmd *cobra.Command) (string, string, string, string, string, string, error) {
```

Comment:
after 3 output vars, I usually revert to making a struct because I forget the ordering of the variables, especially when they are all of the same type. It might make it a little cleaner to do something like this:

```suggestion
type FoggProject struct {
Project, Region, Bucket, Table, Profile, Owner string
}
func userPrompt(cmd *cobra.Command) (*FoggProject, error) {
```

and then in error cases you'd do something like 

~~~go
if err != nil {
  return nil, errors.Wrap(err, "a summary of the error")
}
~~~